### ISSUE TITLE
Bug: Vagrant fails at very early stage after updating to 3.13.2

### ISSUE THREAD

**Issue by luuptekman** [Varying-Vagrant-Vagrants/VVV#2726]
### Are you using the latest stable or develop branch version of VVV?

Yes (stable)

### Is it a new VVV, or an existing VVV that used to work?

Existing, worked but now broken

### Did you use a CustomFile?

No (default)

### Whats the problem?

I used `git pull` to update to latest stable version: 3.13.2.

After that I'm unable to start any vagrant commands such as `vagrant up`, `vagrant ssh` etc....Execution ends with the following error:

```
➜  vvv git:(stable) ✗ vagrant ssh
Vagrant failed to initialize at a very early stage:

There was an error loading a Vagrantfile. The file being loaded
and the error message are shown below. This is usually caused by
a syntax error.

Path: /Users/myuser/dev/vvv/Vagrantfile
Line number: 45
Message: Errno::ENOENT: No such file or directory - docker
```

I noticed this change in Vagrantfile: https://github.com/Varying-Vagrant-Vagrants/VVV/commit/e10bc480339fe5ffb3a8893a5c89e3260131853a#

After modifing line 45 back to `if system("docker version")` vagrant works again.

About my laptop / vagrant:
- Macbook Pro with Apple chip M3
- Vagrant version 2.4.0
- using Parallels

### What is the output of `vagrant status`

```shell
➜  vvv git:(stable) ✗ vagrant status
__ __ __ __
\ V\ V\ V / v3.13.2 Ruby:3.1.4, Path:"/Users/mrluuptek/dev/vvv"
 \_/\_/\_/  git::stable(e0dad5de)

Vagrant failed to initialize at a very early stage:

There was an error loading a Vagrantfile. The file being loaded
and the error message are shown below. This is usually caused by
a syntax error.

Path: /Users/mrluuptek/dev/vvv/Vagrantfile
Line number: 45
Message: Errno::ENOENT: No such file or directory - docker
```

### Which Operating System are you using?

Apple MacOS (Arm64/Apple Silicon)

### Which provider are you using?

Parallels (Apple Silicon/Arm)

**Comment by tomjn** [comment]
Please switch to the develop branch

**Comment by tomjn** [comment]
The `develop` branch swaps the check for docker around to instead check for parallels. Of course installing docker and making sure you have parallels set as the provider in `config.yml` would also fix this ( make sure you set it in `config.yml` anyway ).

It turns out checking that a command exists in ruby isn't so straight forward, `system` sort of works but prints out the result and we can't silence it without doing OS specific things like piping to dev null. backticks fails in the way you've described.

The frustrating part is I need docker on my machines for other things I work with so I can't uninstall it to test this particular issue reliably

**Comment by luuptekman** [comment]
Thanks for reaching out Tom, seems that develop branch has no issues 👍

**Comment by tomjn** [comment]
Closing this out as `develop` has a fix and 3.14 is nearing release
